test/e2e/container_inspect_test.go

```diff
@@ -14,7 +15,6 @@ import (
 )
 
 var _ = Describe("Podman container inspect", func() {
-
```

Comment:
unnecessary whiteline change (not blocking but best to avoid changes these unrelated things)

Reply:
Sorry, auto format did it, and I missed it.